user-interface/src/main/java/life/qbic/datamanager/views/projects/project/samples/registration/batch/SampleBatchInformationSpreadsheet.java

```diff
@@ -212,7 +212,11 @@ public String getSampleLabel() {
     public void setSampleLabel(String sampleLabel) {
       this.sampleLabel = sampleLabel;
     }
-    public String getOrganismId() { return organismId; }
```

Comment:
please revert the code styling changes in this class

Reply:
...for clarity

Reply:
I reformated the file according to google styling and removed the custom linebreaks